### ISSUE TITLE
Can we remove the "Pricing" Tab?

### ISSUE THREAD

**Issue by praveenscience** [killerraj369/Resume-Maker#17]
I was wondering since this is an Open Source project, should we drop the "Pricing" nav bar element from the nav bar? Or is it intentional.

**Comment by killerraj369** [comment]
Yes You can update the Navbar and elements with appropriate fields. There are no pages associated with those elements till now.
Also try to improve the Navbar Like adding github fields, section fields etc.
